package.json

```diff
@@ -14,8 +14,10 @@
   },
   "dependencies": {
     "@sentry/browser": "^5.5.0",
+    "apollo-boost": "^0.4.3",
```

Comment:
Could you check the perf aspects of including `apollo-boost` in the boot time?

Reply:
Maybe [graphql-request](https://github.com/prisma/graphql-request) would be an better alternative because it has a minimal footprint.

Reply:
Used Fetch API to query graphql endpoint